indico/modules/events/surveys/forms.py

```diff
@@ -43,6 +43,21 @@ class SurveyForm(IndicoForm):
     submission_limit = IntegerField(_("Capacity"),
                                     [HiddenUnless('limit_submissions'), DataRequired(), NumberRange(min=1)],
                                     description=_("Maximum number of submissions accepted"))
+    notifications_enabled = BooleanField(_('Notifications'), widget=SwitchWidget(),
+                                         description=_('Whether users are to be notified on specific events within the '
+                                                       'survey'))
+    include_participants_registrants = BooleanField(_('Participants/Registrants'), [HiddenUnless('notifications_enabled')],
+                                                    widget=SwitchWidget(), description=_('Include participants/registrants '
```

Comment:
also, we usually start the `description` kwarg in a new line to avoid excessive string splitting